openmc/tallies.py

```diff
@@ -1141,16 +1092,14 @@ def get_nuclide_indices(self, nuclides):
 
         cv.check_iterable_type('nuclides', nuclides, str)
 
-        # Determine the score indices from any of the requested scores
-        if nuclides:
-            nuclide_indices = np.zeros(len(nuclides), dtype=int)
-            for i, nuclide in enumerate(nuclides):
-                nuclide_indices[i] = self.get_nuclide_index(nuclide)
-
         # If user did not specify any specific Nuclides, use them all
-        else:
-            nuclide_indices = np.arange(self.num_nuclides)
+        if not nuclides:
+            return np.arange(self.num_nuclides)
 
+        # Determine the score indices from any of the requested scores
+        nuclide_indices = np.zeros(len(nuclides), dtype=int)
```

Comment:
We can already be certain that nuclides is just an iterable of strings, so the following should work
```suggestion
        nuclide_indices = np.zeros_like(nuclides, dtype=int)
```